index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portofolio - Here we go</title>
+  </head>
+  <body>
+    <div id="app">
+      <header class="header">
+        <a href="#"><h2>My Logo</h2></a>
+        <a href="#"
+          ><img src="/img/icons/Union.png" alt="toggle menu icon"
+        /></a>
+      </header>
+
+      <div class="showcase">
+        <h3>I’m Joseph Glad to see you!</h3>
```

Comment:
- [ ] This would be more semantic as an h1 tag, kindly change it, as using the correct header tags is very important, especially for screen readers.